implementation/src/main/java/io/smallrye/mutiny/helpers/Subscriptions.java

```diff
@@ -229,6 +229,32 @@ public static long multiply(long n, long times) {
         return u;
     }
 
+    /**
+     * Atomically requests from the Subscription in the field if not null, otherwise accumulates
+     * the request amount in the requested field to be requested once the field is set to non-null.
+     * 
+     * @param field the target field that may already contain a Subscription
+     * @param requested the current requested amount
+     * @param n the request amount, positive (verified)
+     */
+    public static void deferredRequest(AtomicReference<Subscription> field, AtomicLong requested, long n) {
```

Comment:
Should `n` be renamed to `requests`? `Subscriptions.add()` uses requests, but others seem to use `n`

Reply:
I've renamed them.
Sometimes it was not `requests` but `emitted` ;-)